PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskIDFragmentationFunction.cxx

```diff
@@ -765,7 +765,8 @@ Bool_t AliAnalysisTaskIDFragmentationFunction::FillHistograms()
   
   fESD = dynamic_cast<AliESDEvent*>(InputEvent());
   if(!fESD){
-    if(fDebug>3) Printf("%s:%d ESDEvent not found in the input", (char*)__FILE__,__LINE__);
+    if(fDebug>3) 
+      Printf("%s:%d ESDEvent not found in the input", (char*)__FILE__,__LINE__);
```

Comment:
In general I recommend you to use the AliLog facility. In this case you would do
``` c++
AliDebug(3, "ESD Event not found in input");
```
and you can steer this in the macro / ROOT prompt via
``` c++
AliLog::SetClassDebugLevel("AliAnalysisTaskIDFragmentationFunction");
```
Similar loggers exist for error messages as warning (AliWarning), error (AliError) and fatal (AliFatal), the later will terminate execution.

In many cases it now makes more sense to use the c++ streaming facility, particularly if one wants to stream objects where the operator<< is implemented. With AliLog this looks like
``` c++
AliDebugStream(3) << "ESD Event not found in input" << std::endl;
```
(of course using the same debug levels as AliDebug). O2 will use a similar mechanism, called FairLogger.

Reply:
Yes, I know about AliDebug. The class is sprinkled with this old logging facility, I will remove it in another PullRequest all together.